Tutorials/Python_Developer_Tutorials/foundation/Commanding-Local/Proxy-Local-Commands-Tutorial_.py

```diff
@@ -1,87 +1,110 @@
 # This example will show how to command Faraday using the basic command application.
-# LED's being commanded may be recieving other commands and not work as intended (i.e. RED due to RF TX indication)
+# LED's being commanded may be receiving other commands and not work as intended (i.e. RED due to RF TX indication)
 
-#Imports - General
+# Imports - General
 
-import os, sys
-sys.path.append(os.path.join(os.path.dirname(__file__), "../../../../Faraday_Proxy_Tools")) #Append path to common tutorial FaradayIO module
+import os
+import sys
+import time
+import ConfigParser
+
+sys.path.append(os.path.join(os.path.dirname(__file__),
+                             "../../../../Faraday_Proxy_Tools"))  # Append path to common tutorial FaradayIO module
 
-#Imports - Faraday Specific
+# Imports - Faraday Specific
 from FaradayIO import faradaybasicproxyio
 from FaradayIO import faradaycommands
 from FaradayIO import gpioallocations
 
-import time
-
-#Definitions
+# Open configuration INI
+config = ConfigParser.RawConfigParser()
+filename = os.path.abspath("configuration.ini")
+config.read(filename)
 
+# Definitions
 
-#Variables
-local_device_callsign = 'REPLACEME'  # Should match the connected Faraday unit as assigned in Proxy configuration
-local_device_node_id = REPLACEME  # Should match the connected Faraday unit as assigned in Proxy configuration
+# Variables
+local_device_callsign = config.get("DEVICES",
+                                   "UNIT0CALL")  # Should match the connected Faraday unit as assigned in Proxy configuration
+local_device_node_id = config.get("DEVICES",
```

Comment:
might want to use .getint()

https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.getint